rust/src/graph/direct_import_queries.rs

```diff
@@ -54,4 +55,58 @@ impl Graph {
             None => &EMPTY_IMPORT_DETAILS,
         }
     }
+
+    pub fn find_matching_direct_imports(
+        &self,
+        importer_expression: &ModuleExpression,
+        imported_expression: &ModuleExpression,
+    ) -> FxHashSet<(ModuleToken, ModuleToken)> {
```

Comment:
🍷 `(ModuleToken, ModuleToken)` could probably be made a bit nicer with a struct, but it's consistent with what we're doing in the rest of the rust code for now, so I think I'd leave it like this here and leave the introduction of that struct to a separate change.